resource-timing/resource_TAO_multi.htm

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8" />
+<title>Resource Timing TAO tests</title>
+<link rel="author" title="Google" href="http://www.google.com/" />
+<link rel="help" href="https://www.w3.org/TR/resource-timing-2/#timing-allow-origin"/>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="resources/webperftestharness.js"></script>
+<script src="resources/webperftestharnessextension.js"></script>
+<script>
+setup({explicit_done: true});
+
+// explicitly test the namespace before we start testing
+test_namespace("getEntriesByType");
+
+var d;
+var iframe;
+var iframeBody;
+var image;
+var random = Math.random();
+
+function setup_iframe() {
+    iframe = document.getElementById('frameContext');
+    d = iframe.contentWindow.document;
+    iframeBody = d.body;
+    iframe.addEventListener('load', onload_test, false);
+}
+function onload_test() {
+    if (window.performance.getEntriesByType === undefined) {
+      done();
+      return;
+    }
+    var context = new PerformanceContext(iframe.contentWindow.performance);
+    var entries = context.getEntriesByType('resource');
+
+    if(entries.length > 0) {
+        entry = entries[0];
+
+        test_equals((entry.redirectStart + entry.redirectEnd + entry.domainLookupStart + entry.domainLookupEnd + entry.connectStart + entry.connectEnd + entry.secureConnectionStart + entry.requestStart + entry.responseStart), 0, 'redirectStart, redirectEnd, domainLookupStart, domainLookupEnd, connectStart, connectEnd, secureConnectionStart, requestStart, and responseStart -- should be all returned as 0 when the HTTP response includes more than one Timing-Allow-Origin header value and TAO algorithm fails, if not overwrite any existing TAO values when appending the new value');
+    }
+
+    done();
+}
+window.setup_iframe = setup_iframe;
+</script>
+</head>
+<body>
+<h1>Description</h1>
+<p>This test validates that for a cross origin resource, the timing allow check algorithm will fail when the HTTP response includes more than one Timing-Allow-Origin header value (not overwrite any existing TAO values when appending the new value).</p>
```

Comment:
L1 allowed a space separated list of origins, which in theory meant that "A.com B.com" is a valid TAO header-value that should cause TAO checks to pass for same-origin resources, and any resources from A.com and B.com. In practice, this was inconsistently implemented by browsers and, as far as we can tell, never widely adopted by developers.

L2 changed the above definition to be a comma separated list, meaning that "A.com B.com" is now an invalid value, instead developers should specify "A.com, B.com" OR provide multiple TAO headers, which are implicitly concatenated into a comma separated list by the UA (this is generic header value treatment per HTTP spec).

As far as tests, I think we want: 
- Test explicit comma separated list - i.e. "TAO: A.com, B.com"
- Test concatenated list - i.e. "TAO: A.com" + "TAO: B.com"
- Test that "*" value matches all origins
- Test that space separate list fails: "TAO: A.com B.com" should not pass for either A.com or B.com